reframe/frontend/cli.py

```diff
@@ -618,7 +613,41 @@ def print_infoline(param, value):
             exec_policy.sched_reservation = options.reservation
             exec_policy.sched_nodelist = options.nodelist
             exec_policy.sched_exclude_nodelist = options.exclude_nodes
-            exec_policy.sched_options = options.job_options
+            if options.account:
```

Comment:
All these deprecation warnings should go a bit up, because now they appear only when you try to run the tests. They should appear always, even when you are listing them.